File: week-01/day-05/algo-pad-array/pad_array.py
#Returns input array with value appended to array until the
#array's length is equal to minSize
def pad(array, min_size, value = None):

    #Copy array and add value to the copy until the length is minSize
    newArray = array.copy()
    while len(newArray) < min_size:
        newArray.append(value)

    return newArray #Return padded array copy

File: week-01/day-05/algo-pad-array/padArray.js
//Returns input array with value appended to array until the
//array's length is equal to minSize
exports.pad = function(array, minSize, value = null) {

    //Copy array and add value to the copy until the length is minSize
    let newArray = array.slice();
    while (newArray.length < minSize) newArray.push(value);

    return newArray; //Return padded array copy
}